src/pages/cli/graphql/override.mdx

```diff
@@ -76,6 +76,48 @@ export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
   resources.models["Todo"].modelDatasource.dynamoDbConfig['deltaSyncConfig']['baseTableTtl'] = '3600'
 }
 ```
+### Example - Configure DynamoDBBillingMode
+
+Set the DynamoDB billing mode for the API. One of "PROVISIONED" or "PAY_PER_REQUEST".
```

Comment:
```suggestion
Set the [DynamoDB billing mode](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_BillingModeSummary.html) for the API. One of "PROVISIONED" or "PAY_PER_REQUEST".
```

since we link to Point-in-time-recovery I think it would be helpful to link each of these properties we are overriding